lib/takeaway.rb

```diff
@@ -0,0 +1,52 @@
+require './lib/restaurant'
+
+class TakeAway
+  attr_reader :restaurant, :dish, :basket
+
+  def initialize (restaurant = Restaurant.new)
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)